assets/sass/bulma_styles.scss

```diff
@@ -1,2 +1,26 @@
 @charset "utf-8";
+
+/* GLOBAL VARIABLES */
+$hyper: #3A10E5;
+$white: #FFFFFF;
+$beige: #FFF0E5;
+$primary: $beige;
+
+$body-background-color: $beige;
+$link: black;
+$link-hover: $hyper;
+
+
+/* NAVBAR VARIABLES */
+$navbar-height: 5rem;
+$navbar-background-color: $primary;
+$navbar-box-shadow-size: 0 1px 2px 1px;
+$navbar-box-shadow-color: grey;
```

Comment:
You know, this looks good, but I kinda liked your original navbar shadow better.  What do you think?

Reply:
I've reset it to the old one 🙂